xarray/coding/cftime_offsets.py

```diff
@@ -91,9 +91,9 @@ class BaseCFTimeOffset:
     _day_option: ClassVar[Optional[str]] = None
 
     def __init__(self, n=1):
```

Comment:
Can type the `n` in the init as well now that we're touching things here anyway:
`def __init__(self, n: int | float = 1)`